src/app/pages/home/home.component.ts

```diff
@@ -9,39 +9,8 @@ import { Router } from '@angular/router';
 
 export class HomeComponent implements OnInit {
     posts: object;
-    post:{
-        id: number,
-        title: string,
-        username:string,
-        description: string,
-        photo: string,
-        post_status: string,
-        post_priority: string,
-        city: string,
-        state: string,
-        zip_code: string,
-        email: string,
-        customer_id:number,
-        customerId:string,
-        can_bid: boolean,
-        budget: number
-      } = {
-        id:-1,
-        title:'',
-        description:'',
-        username:'',
-        photo:'',
-        post_status:'',
-        post_priority:'',
-        city:'',
-        state:'',
-        zip_code:'',
-        email:'',
-        customer_id:null,
-        customerId:null,
-        can_bid:false,
-        budget:null
-      }
+    post: object;
```

Comment:
Again, don't throw away the work you already had. You should be updating that typescript definition rather than just setting this to an object.

Reply:
Jen has resolved this issue on her branch